mixer/adapter/solarwinds/config/BUILD

```diff
@@ -0,0 +1,28 @@
+# gazelle:exclude config.pb.go
+
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+load("@org_pubref_rules_protobuf//gogo:rules.bzl", "gogoslick_proto_library")
+
+gogoslick_proto_library(
```

Comment:
please do not add this to the repo.  use `go:generate` style generation moving forward.

Reply:
done